sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -363,6 +363,17 @@ def __init__(self, options):
       self.proto.type = dataflow.Job.TypeValueValuesEnum.JOB_TYPE_STREAMING
     else:
       self.proto.type = dataflow.Job.TypeValueValuesEnum.JOB_TYPE_BATCH
+
+    # Labels
+    if self.google_cloud_options.labels:
+      self.proto.labels = dataflow.Job.LabelsValue()
+      for label in self.google_cloud_options.labels:
+        label = label.split('=')
```

Comment:
<!--thread_id:cc_144458604_t; commit:a4e36ef94c4f7a26c96e424c18408b5ae42ca16d; resolved:1-->
<!--section:context-quote-->
> **charlesccychen** wrote:
> Let's not reuse `label`; how about `parts = label.split('=', 1)`?

<!--section:body-->
Done. Thank you.